src/MergeContext.js

```diff
@@ -198,6 +218,11 @@ class MergeContext {
             return false;
         }
 
+        await this._checkApproved();
+        this._log("_checkApproved: " + this._approvalResult.description + ", state: " +
```

Comment:
Please move this "serialization" into Approval::toString() so that it is called automatically and can easily be more flexible (e.g., does not have to print nil delays).